lambdas/check-existing-identity/src/test/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandlerTest.java

```diff
@@ -815,46 +760,13 @@ void shouldResetIdentityIfDataDoesNotCorrelateAndNotF2F() {
         assertNull(ipvSessionItem.getVot());
     }
 
-    @Test
-    void shouldReturn500IfFailedToParseCredentials() throws Exception {
```

Comment:
Doesn't need to parse credentials since they are VerifiableCredentials and are already valid (this change is repeated in a number of places)